packages/schematics/migrations/update-6-1-1/update-6-1-1.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  Rule,
+  Tree,
+  SchematicContext,
+  chain,
+  template,
+  apply,
+  mergeWith,
+  url
+} from '@angular-devkit/schematics';
+import { updateJsonInTree, readJsonInTree } from '../../src/utils/ast-utils';
+import { stripIndents } from '@angular-devkit/core/src/utils/literals';
+
+function displayInformation(_, context: SchematicContext) {
+  context.logger.info(stripIndents`
+    A global base karma config has been added at karma.conf.ts
+    
+    This file exports a karma config to be extend in each project
+  `);
+
+  context.logger.warn(stripIndents`
+    This new file is not being used yet!
+
+    Generate a new project to see an example of how it might be used.
+  `);
+}
+
+function addGlobalKarmaConf() {
+  const templateSource = apply(url('./files'), [
+    template({
+      tmpl: ''
+    })
+  ]);
+  return mergeWith(templateSource);
+}
+
+function renameExistingKarmaConfs(host: Tree, context: SchematicContext) {
```

Comment:
I think we need to stick JS. 

a) This is what the CLI generates
b) It makes our schematics simpler (no need to update angular.json)
c) we don't have an appropriate tsconfig to compile karma files